src/package_io/constants.star

```diff
@@ -42,3 +42,5 @@ ZERO_ADDRESS = "0x0000000000000000000000000000000000000000"
 
 # 256-bit zero hash.
 ZERO_HASH = "0x0000000000000000000000000000000000000000000000000000000000000000"
+
+AGGKIT_BINARY_NAME = "aggkit"
```

Comment:
This constant seems specific to the aggkit so I think it should be defined in `lib/cdk_node.star` instead.